packages/client/src/api/discover.ts

```diff
@@ -14,15 +15,13 @@ const OIDCConfigurationSchema = s.type({
 
 export type OIDCConfiguration = s.Infer<typeof OIDCConfigurationSchema>;
 
-const appendSlashIfNeeded = (url: string): string => {
-  if (url.endsWith('/')) {
-    return url;
-  }
-
-  return url + '/';
-};
-
-export default async function discover(
+/**
+ * OIDC Connect Discovery: https://openid.net/specs/openid-connect-discovery-1_0.html
```

Comment:
a potentially thrown exceptions should be documented ?
a `LogtoError` may be thrown ?